src/components/dialogs/network-modifications/two-windings-transformer/creation/two-windings-transformer-creation-dialog.js

```diff
@@ -444,9 +444,15 @@ const TwoWindingsTransformerCreationDialog = ({
                         twt?.[PHASE_TAP_CHANGER]
                     ),
                     currentLimiterRegulatingValue:
```

Comment:
The method getPhaseTapChangerFormData is also called line 266, maybe you should do the change there too ?

Reply:
Indeed. Even if it is much less likely to happen here, it will be cleaner. Done.